product_code_mandatory/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
please use the new split readme [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0) and generate readme from it.

Reply:
fragmented readme, bot will generate the readme file